website/api/controllers/webhooks/receive-from-github.js

```diff
@@ -338,76 +340,57 @@ module.exports = {
           // For each changed file, decide what reviewer to request, if any…
           for (let changedPath of changedPaths) {
             changedPath = changedPath.replace(/\/+$/,'');// « trim trailing slashes, just in case (b/c otherwise could loop forever)
-            sails.log.debug(`…checking DRI of changed path "${changedPath}"`);
+            sails.log.verbose(`…checking DRI of changed path "${changedPath}"`);
 
             let reviewer = undefined;//« whether to request review for this change
             let exactMatchDri = DRI_BY_PATH[changedPath];
-            if (exactMatchDri) {
-              let isAuthorDRI = exactMatchDri === issueOrPr.user.login.toLowerCase();//« See `user.login` in https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#get-a-pull-request
-              let isSenderDRI = exactMatchDri === sender.login.toLowerCase();
-              if (isAuthorDRI || isSenderDRI) {
-                // If the original PR author OR you, the sender (current PR author/editor) are the DRI,
-                // then do nothing.  No need to request review from yourself, and you CAN'T request
-                // review from the author (or the GitHub API will respond with an error.)
-              } else {
-                // Otherwise, we've found our match.  We'll request review from this person.
-                // (And we'll stop looking.)
-                reviewer = exactMatchDri;
-              }
+            if (exactMatchDri) {// « If we've found our DRI, then we'll stop looking (for *this* changed path, anyway)
+              reviewer = exactMatchDri;
```

Comment:
pulling out the isAuthor and isSender checks allows this to be reliable for correctly labeling/unlabeling with the `#g-ceo` label, which also makes sure pending PRs waiting on me appear in the kanban board Sam and I are using to track them